CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
    instead of using Thread.raise on the plugins' threads. Ref: https://github.com/elastic/logstash/pull/3895
  - Dependency on logstash-core update to 2.0
 
+# v 0.1.4
```

Comment:
In master, as we introduced some stop breaking changes, not much affecting filters, but we raised all versions to be ```2.x```, we should align with this versions. Check the ```.gemspec``` for more details.